workshop/06_deployment_statefulset_daemonset/statefulset.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Fully functional RabbitMQ cluster (3 replicas) to demonstrate OrderedReady rollout strategy